docs/apache-airflow-providers-google/operators/cloud/dataplex.rst

```diff
@@ -142,3 +141,152 @@ To delete a lake you can use:
     :dedent: 4
     :start-after: [START howto_dataplex_delete_lake_operator]
     :end-before: [END howto_dataplex_delete_lake_operator]
+
+Create a data quality
+---------------------
+
+Before you create a dataplex data quality you need to define its body.
+For more information about the available fields to pass when creating a data quality, visit `Dataplex create data quality API. <https://cloud.google.com/dataplex/docs/reference/rest/v1/projects.locations.dataScans#DataScan>`__
+
+A simple data quality configuration can look as followed:
+
+.. exampleinclude:: /../../tests/system/providers/google/cloud/dataplex/example_dataplex_dq.py
+    :language: python
+    :dedent: 0
+    :start-after: [START howto_dataplex_data_quality_configuration]
+    :end-before: [END howto_dataplex_data_quality_configuration]
+
+With this configuration we can create the data quality:
+
+:class:`~airflow.providers.google.cloud.operators.dataplex.DataplexCreateDataQualityScanOperator`
+
+.. exampleinclude:: /../../tests/system/providers/google/cloud/dataplex/example_dataplex_dq.py
+    :language: python
+    :dedent: 4
+    :start-after: [START howto_dataplex_create_data_quality_operator]
+    :end-before: [END howto_dataplex_create_data_quality_operator]
+
+Delete a data quality
+---------------------
+
+To delete a data quality you can use:
+
+:class:`~airflow.providers.google.cloud.operators.dataplex.DataplexDeleteDataQualityScanOperator`
+
+.. exampleinclude:: /../../tests/system/providers/google/cloud/dataplex/example_dataplex_dq.py
+    :language: python
+    :dedent: 4
+    :start-after: [START howto_dataplex_delete_data_quality_operator]
+    :end-before: [END howto_dataplex_delete_data_quality_operator]
+
+Run a data quality
+------------------
+
+We have possibility to run Dataplex data quality in the sync mode and async:
```

Comment:
```suggestion
You can run Dataplex Data Quality in asynchronous modes to later check its status using sensor:
```
The example yo are pointing to is running asynchronously only

Reply:
changed